xml/decoders/SoundTraxx_Tsu2_Diesel.xml

```diff
@@ -134,12 +140,12 @@
             <date>2020-09-17</date>
             <authorinitials>MJM</authorinitials>
             <revremark>Fix sound select for PNP8 and 21PNEM8 decoders</revremark>
-        </revision>
-        <revision>
-            <revnumber>12</revnumber>
-            <date>2021-02-08</date>
-            <authorinitials>MJM</authorinitials>
-            <revremark>Fix ID for PNP8 and 21PNEM8 decoders</revremark>
+        </revision> 
+		    <revision>
+            <revnumber>13</revnumber>
+            <date>2021-03-19</date>
+            <authorinitials>MFO</authorinitials>
+            <revremark>Added TSU-KN1 decoder</revremark>
         </revision>
```

Comment:
Please keep the info about revision 12, but change it to revision 13. And change your revision to revision 14.
@mmosher5501 did that change in PR #9492, but missed to give it revision 13.

Reply:
Correction was done, but had no idea what Michael did in his so the revremark is not aaccurate . 
I believe he made  changes to the 'includes' within the definition.  These were not in JMRI 4.22

Reply:
Thanks.